.env.example

```diff
@@ -0,0 +1,2 @@
+REDIS_HOST=<HOST_HERE>
```

Comment:
I do not see much value in in. I'd rather have exemplar file with values for local environment or development environment, something which has no secret secrets.